app/Console/Kernel.php

```diff
@@ -132,13 +132,12 @@ private function scheduleJobsForCollections(Schedule $schedule): void
             ->everyThirtyMinutes();
 
         // Fetch banners for collections that don't have one yet
-        // (More often according to config)
         $schedule
             ->command(FetchCollectionBanner::class, [
                 '--missing-only',
             ])
             ->withoutOverlapping()
-            ->dailyAt('5:30');
+            ->weekly();
```

Comment:
I'd keep this at daily as it's only for missing banners, which makes sense to try and fetch more often as new collections may add one shortly after we fetch them